site/README.md

```diff
@@ -20,7 +20,3 @@ When the langauge is `jsx` and the code block has a `// @live` comment, it will
 render('some react element')
 \`\`\`
 ```
-
-### Links
-
-All links to other pages on the docs should be linked to with `https://emotion.sh/docs/doc-name`, these links are transformed so they are local links.
```

Comment:
q: how this has changed?

Reply:
Hm,, I'm not sure what this refers to **exactly** - maybe it was misworded or something. One thing I know that was being rewritten were relative GitHub links - so they were working, here, on GitHub, and on the deployed site (where the structure was slightly different).

Maybe @mitchellhamilton would know more about this note here

Reply:
I think it was so that you could have links in READMEs that would work when viewing the README on npm